src/translations/en.json

```diff
@@ -352,6 +352,7 @@
     "volume_normalization": "Volume normalization",
     "play_now_replace": "Play Now (clear queue)",
     "currently_playing": "Currently playing",
+    "currently_queued": "Currently queued",
```

Comment:
I would really change this to currently playing

Reply:
Fair enough, will do